src/main/webapp/index.html

```diff
@@ -32,6 +32,43 @@ <h1 class="index-registration-block-child">TravIS: Travel Information System</h1
         </form>
       </div>
       <div id="index-start-trip-block">
+        <form action="/calculate-trip" method="POST" id="startTripForm">
+          <div class="form-row">
+            <div class="form-group col-md-3">
+              <input type="text" placeholder="Destination" class="form-control" 
+                name="inputDestination" id="inputDestination">
+            </div>
+            <div class="form-group col-md-3">
+              <input type="text" placeholder="Start Date" class="form-control" 
+                name="inputStartDate" id="inputStartDate">
+            </div>
+            <div class="form-group col-md-3">
+              <input type="text" placeholder="End Date" class="form-control is-valid" 
+                name="inputEndDate" id="inputEndDate" readonly>
+            </div>
+            <div class="form-group">
+              <button type="button" class="btn btn-secondary">Next</button>
+            </div>
+          </div>
+          <div class="form-row">
+            <div class="form-group col-md-3">
+              <input type="text" placeholder="Point of Interest" 
+                class="form-control" id="inputPoi">
+            </div>
+            <div class="form-group">
+              <button type="button" class="btn btn-primary">Add POIs</button>
+            </div>
+            <div class="form-group">
+              <input type="button" value="California (click to remove)" 
```

Comment:
I would probably have an 'x' or edit ability, rather than have "(click to remove)" on the button. But since this is so early, you don't need to do anything about this right now. Just wanted to address it.

Reply:
I agree with that as well -- I tried to add an "X" button for removal at first, but the design was a bit difficult so I stuck with this. Once we finish the prototype, I'll be sure to update this!